std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -207,7 +162,7 @@ public static LocalTime parse_time(String text, String pattern, Locale locale) {
    * types is not supported, so this is a workaround.
    *
    * <p>TODO once the related issue is fixed, this workaround may be replaced with pattern matching
-   * in Enso; the related Pivotal issue: https://www.pivotaltracker.com/story/show/183219169
+   * in Enso; <a href="https://www.pivotaltracker.com/story/show/183219169">Pivotal issue.</a>
```

Comment:
If we are modifying this part then:
1. firstly, let's update the link to GH as the Pivotal one seems dead: https://github.com/enso-org/enso/issues/4597
2. ideally, we could remove this workaround and use the pattern matching, since the task has been completed ages ago. But that's lower priority I guess.